src/main/java/core/basesyntax/FigureBehaviour.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureBehaviour {
```

Comment:
Возможность быть нарисованным - это не специфическое свойство Фигуры. Этот интерфейс могут имплементировать разные классы.
```suggestion
public interface Drawable {
```